CHANGES

```diff
@@ -3,14 +3,64 @@ Release 1.15 (development)
 
 .. rubric:: Enhancements
 
+* Support sending a ``RAUC-System-Version`` HTTP header in the initial
+  streaming/inspect request.
+* When using the service, send `custom HTTP headers
+  <https://rauc.readthedocs.io/en/latest/reference.html#handlers-section>`_ set
+  by the ``system-info`` handler for ``rauc info``/``InspectBundle()`` as well.
+* Set up groups of the streaming helper process to allow access to required
```

Comment:
For clarity, maybe `Set up group memberships of the streaming helper process..`?